apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SessionPoolOptions.cs

```diff
@@ -22,6 +23,24 @@ namespace Google.Cloud.Spanner.V1
     public sealed class SessionPoolOptions
     {
         /// <summary>
+        /// Constructs a new <see cref="SessionPoolOptions"/>.
+        /// </summary>
+        public SessionPoolOptions()
+        {
+            var retry = SpannerSettings.GetDefault().CommitSettings.Timing.Retry;
+            var timeout = retry?.TotalExpiration.Timeout;
+            if (timeout.HasValue)
+            {
+                Timeout = (int)timeout.Value.TotalSeconds;
```

Comment:
which rpcs is this timeout used for?

Reply:
in ado.net there is no way to distinguish since the timeout is at the command level.
So this timeout will be used for commit, but also for session create, rollback and begin transaction.